packages/hasura-auth-js/src/hasura-auth-client.ts

```diff
@@ -79,6 +79,8 @@ export class HasuraAuthClient {
   readonly url: string
   constructor({
     url,
+    subdomain,
```

Comment:
ins't this technically a breaking change? maybe we can add a broadcast key that is optional and then set it from the main constructor to the sane $subdomain$region value